cpp/ql/test/experimental/query-tests/Security/CWE/CWE-190/DangerousUseOfTransformationAfterOperation/test.cpp

```diff
@@ -0,0 +1,23 @@
+void testCall (unsigned long);
+void functionWork() {
+  unsigned long aL;
+  char aA[10],*aP;
+  unsigned char aUC;
+  int aI;
+  unsigned int aUI;
+  aI = (aUI*8)/10; // GOOD
```

Comment:
This is using unitialized memory (which is undefined behavior). Could you take `aUI` as a parameter instead (or initialize it)?

Similarly, some of the other expressions also use uninitialized memory. Please initialize those or take them as parameters instead :)